doc/metadata_routing.rst

```diff
@@ -61,88 +83,76 @@ in this section require the following imports and data::
 Weighted scoring and fitting
 ----------------------------
 
-Here :class:`~model_selection.GroupKFold` requests ``groups`` by default. However, we
-need to explicitly request weights for our scorer and the internal cross validation of
-:class:`~linear_model.LogisticRegressionCV`. Both of these *consumers* know how to use
-metadata called ``sample_weight``::
+:class:`~model_selection.GroupKFold` requests ``groups`` by default. However, we need to
+explicitly request weights for our scorer and the internal cross validation of
+:class:`~linear_model.LogisticRegressionCV` by specifying `sample_weight=True`. Both of
+these :term:`consumers <consumer>` know how to use metadata called ``sample_weight``::
 
-  >>> weighted_acc = make_scorer(accuracy_score).set_score_request(
-  ...     sample_weight=True
-  ... )
-  >>> lr = LogisticRegressionCV(
-  ...     cv=GroupKFold(), scoring=weighted_acc,
-  ... ).set_fit_request(sample_weight=True)
+  >>> weighted_acc = make_scorer(accuracy_score).set_score_request(sample_weight=True)
+  >>> lr = LogisticRegressionCV(cv=GroupKFold(), scoring=weighted_acc,)
+  >>> lr.set_fit_request(sample_weight=True)
```

Comment:
why the format change? it's fine as is, isn't it?

Reply:
I had hoped for the longest possible lines, but now I got aware that black is not formatting rst files.

Though, I still think the old formatting it's complicated to read. Let me try some compromise and please let me know what you think.